posts/Yoann/2018-02-07-hibernate-search-5-9-0-Final.adoc

```diff
@@ -0,0 +1,97 @@
+for an introduction to JSR 352 in Hibernate Search.
+
+[[wildfly-feature-packs]]
+=== JBoss modules now distributed as feature packs
+
+The Hibernate Search JBoss Modules, for use in WildFly or JBoss EAP,
+are now distributed as Wildfly feature packs, which makes provisioning
+a WildFly server with an upgraded Hibernate Search version easier than ever.
+The feature packs can also be consumed using http://wildfly-swarm.io/[WildFly Swarm].
+
+Also, the modules are now distributed as *multiple* feature packs,
+which means for example that you won't need to add the Elasticsearch clients
+to your server if you only use Lucene.
+
+See https://docs.jboss.org/hibernate/search/5.9/reference/en-US/html_single/#search-configuration-deploy-on-wildfly[the documentation]
+for more information on the available feature packs and how to consume them.
+
+=== Other changes
+
+Here are the other notable changes since Hibernate Search 5.8.2.Final:
+
+* https://hibernate.atlassian.net/browse/HSEARCH-2832[HSEARCH-2832]:
+Exception messages when an Elasticsearch work fails are now a bit less
+verbose. In particular they don't include the failing request anymore,
+because it can often be huge and ends up flooding the logs (especially due to request bulking).
+You can still log every Elasticsearch request
+https://docs.jboss.org/hibernate/search/5.9/reference/en-US/html_single/#elasticsearch-logging[as explained in the documentation].
+* https://hibernate.atlassian.net/browse/HSEARCH-2909[HSEARCH-2909]:
+Annotations in the default package will no longer make the annotation processing fail.
+* https://hibernate.atlassian.net/browse/HSEARCH-2935[HSEARCH-2935]:
+The CPU cost of static initialization of Hibernate Search loggers
+has been sensibly reduced, improving startup performance.
+* https://hibernate.atlassian.net/browse/HSEARCH-1885[HSEARCH-1885]:
+Custom bridges with specific dependencies such as Tika bridges
+can now be used in a modular environment such as WildFly.
+* https://hibernate.atlassian.net/browse/HSEARCH-2955[HSEARCH-2955]:
+Faceting queries will no longer fail when the index is empty
+or only contains document without any value in the faceting field.
+* https://hibernate.atlassian.net/browse/HSEARCH-2983[HSEARCH-2983]:
+Boost is now consistently taken into account when building numeric or null encoding queries with the DSL.
+
+For a full list of changes since 5.8,
+please see https://hibernate.atlassian.net/issues/?jql=project%20%3D%20HSEARCH%20AND%20fixVersion%20in%20(5.9.0.Final%2C%205.9.0.CR1%2C%205.9.0.Beta1)%20ORDER%20BY%20updated[this list of tickets on our JIRA instance].
+
+For a full list of changes since 5.9.0.CR1,
+please see https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10061&version=31630[the release notes].
+
+== How to get this release
+
+All the necessary information is available (and updated regularly)
+on http://hibernate.org/search/releases/5.9/#get-it[hibernate.org].
```

Comment:
When reading this on staging it didn't look like you'd point to the right page as it states "just look on hibernate.org".

Maybe clarify that you're linking to the right page?

   All details are available and up to date on [the dedicated page on hibernate.org].